timemachine/cpp/src/neighborlist.cu

```diff
@@ -64,22 +64,33 @@ void Neighborlist<RealType>::compute_block_bounds_host(
     DeviceBuffer<double> d_coords(N * D);
     DeviceBuffer<double> d_box(D * D);
 
+    std::vector<RealType> h_block_bounds_centers(this->num_column_blocks() * 3);
+    std::vector<RealType> h_block_bounds_extents(this->num_column_blocks() * 3);
+
     d_coords.copy_from(h_coords);
     d_box.copy_from(h_box);
 
     this->compute_block_bounds_device(N, D, d_coords.data, d_box.data, static_cast<cudaStream_t>(0));
     gpuErrchk(cudaDeviceSynchronize());
 
     gpuErrchk(cudaMemcpy(
-        h_bb_ctrs,
+        &h_block_bounds_centers[0],
```

Comment:
nit/question: why `&ptr[0]` instead of just `ptr`?

Reply:
Because `h_block_bounds_center` is a vector and not a pointer. Trying to access the underlying array that the vector contains by doing `&vec[0]`

Reply:
Ah, ok. I missed that the "centers" variable had changed from a raw buffer to a vector.